composer.json

```diff
@@ -14,13 +14,13 @@
     }
   ],
   "require": {
-    "php": ">=5.2",
+    "php": ">=5.4",
```

Comment:
Security support for 5.4 runs out on 14 Sep 2015. Would recommend bumping up to 5.5 now; security support for 5.5 expires 10 Jul 2016.